Framework/DataHandling/inc/MantidDataHandling/LoadBankFromDiskTask.h

```diff
@@ -42,9 +42,9 @@ class MANTID_DATAHANDLING_DLL LoadBankFromDiskTask : public Kernel::Task {
   void prepareEventId(::NeXus::File &file, int64_t &start_event,
                       int64_t &stop_event,
                       const std::vector<uint64_t> &event_index);
-  std::unique_ptr<uint32_t[]> loadEventId(::NeXus::File &file);
```

Comment:
We have to be a little careful in these cases. Returning a pointer vs returning a `vector` by value could incur larger costs if return value optimization doesn't kick in. As these were already using `unique_ptr` can we leave them as they were?